### ISSUE TITLE
Can we assume this project is now dead and won't ever be updated for Ionic 4 / Meteor 1.8.x ?

### ISSUE THREAD

**Issue by rebound-software** [Urigo/Ionic2CLI-Meteor-WhatsApp#196]

**Comment by rebound-software** [comment]
I have a couple of projects in development based on this Ionic 3 and Meteor 1.6 tutorial and have been hoping there might be some sign of movement towards a functioning Ionic 4 and Meteor 1.8.x version. I'm assuming based on the lack of activity that there is no real interest or intention of this happening anytime soon (or ever?)
I'm certainly grateful to all that have contributed and sadly am not in a position to move this forward myself but it would be helpful to myself and probably a fair few other to get some clarity on this so that  I can look for an alternative framework (a real shame to have to do so) ...  anyone?

**Comment by hsta** [comment]
I read your comment and thought I give you a short answer since we work with Meteor and Ionic on some project for a couple of years now and migrated to Ionic 4 already with the early beta.

We started working with the Meteor/Angular stack in Angular 1 times and also based on @Urigo's ideas/repos at that time.
The ideas were great already back than and at the same time his and repos around his projects were also already back than badly maintained and suffered from bad implementations.

So we got rid of nearly all them and since Angular 2. The only package left from the @Urigo's surroundings was and still is the meteor client bundler. And that is so small and simple that we feel confident to maintain it ourselves should the need arise (it also has alternatives).

What's left is Ionic 4 app complete following the Ionic 4's documentation and standards with a global `Meteor` instance. And that works great.

We had one project which also used @Urigo's `meteor-rxjs` and even that turned out to be a bad idea. While again the idea behind the package is nice, having a syntactic sugar to work with Meteor like you would work with any other Angular Observable, there is no value added besides this syntactic sugar. Meteor's minimongo queries allow you to do nearly the same than RxJS Observables do and unfortunately `meteor-rxjs` messes up in real world use cases. I.e. when collections got bigger we ran into frequent app crashes.
So just use Meteor directly and in cases where it is justified wrap it in an RxJS Observable yourself (RxJS is easy in that regard). Now you also can run your change detections where it is useful and efficiently work with Angular's `onPush` strategy.

<br>

**tl;dr** I would recommend staying away from these repos besides their ideas. You will still be able to combine Meteor with nearly every frontend (also Ionic 4) and now can rely on the documentation, build tools and other ecosystem advantages of these frontends alone. I can ensure you it works and is refreshingly simple :) Only cleaning up old existing projects from the @Urigo's repos can be a bit of a pain but it could be worth it.
